recipes/benchmark/all/conanfile.py

```diff
@@ -57,7 +57,8 @@ def _configure_cmake(self):
                 cmake.definitions["HAVE_POSIX_REGEX"] = False
                 cmake.definitions["HAVE_STEADY_CLOCK"] = False
             else:
-                cmake.definitions["BENCHMARK_BUILD_32_BITS"] = "ON" if "64" not in str(self.settings.arch) else "OFF"
+                cmake.definitions["BENCHMARK_BUILD_32_BITS"] = ("OFF" if "64" in str(self.settings.arch)
+                                                                       or 'armv8' == self.settings.arch else "ON")
```

Comment:
should you use same condition as below 
```py
'arm' in self.settings.arch
```

Reply:
armv8 and above supports 64bit.
https://en.wikipedia.org/wiki/ARM_architecture_family#64/32-bit_architecture

maybe I should also add support of armv9 which is the newer release 🤔